library/src/test/java/com/chuckerteam/chucker/TestTransactionFactory.kt

```diff
@@ -0,0 +1,92 @@
+import com.chuckerteam.chucker.internal.data.entity.HttpTransaction
+import java.util.Date
+
+object TestTransactionFactory {
+
+    internal fun createTransaction(method: String): HttpTransaction {
+        return HttpTransaction(
+            id = 0,
+            requestDate = Date(1300000).time,
+            responseDate = Date(1300300).time,
+            tookMs = 1000L,
+            protocol = "HTTP",
+            method = method,
+            url = "http://localhost/getUsers",
+            host = "localhost",
+            path = "/getUsers",
+            scheme = "",
+            responseTlsVersion = "",
+            responseCipherSuite = "",
+            requestContentLength = 1000L,
+            requestContentType = "application/json",
+            requestHeaders = null,
+            requestBody = null,
+            isRequestBodyPlainText = true,
+            responseCode = 200,
+            responseMessage = "OK",
+            error = null,
+            responseContentLength = 1000L,
+            responseContentType = "application/json",
+            responseHeaders = null,
+            responseBody = """{"field": "value"}""",
```

Comment:
KtLint is unhappy of the formatting here. You can run `./gradlew ktlintFormat` before pushing to make sure that your code is formatted properly.